aap-common/assembly-release-notes.adoc

```diff
@@ -0,0 +1,48 @@
+ifdef::context[:parent-context: {context}]
+
+[id="ap-release-notes"]
+
+= Release notes
+
+[discrete]
+== What’s new in Ansible Automation Platform?
+
+Check out the newest features and enhancements we've just released.
+
+This release includes a number of enhancements, additions, and fixes that have been implemented in the {PlatformNameShort}.
+
+[discrete]
+== New features and enhancements
+
+These releases of {AAPonAWS} and {AAPonGCP} of version 2.4.20230630-00 include the following enhancements:
```

Comment:
This release of...

Unless you want GCP changes in the AWS documents, this should be made conditional, like the Command Generator section.